src/App.tsx

```diff
@@ -10,33 +12,31 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const selectedTab: Tab = tabs.find(tab => (
+    tab.id === selectedTabId
+  )) || tabs[0];
```

Comment:
Move this function outside the component to `helpers` as you need it in Tabs component (read the next comment)

Reply:
ops, yesterday i didn't save changes